docs/website/docs/plus/getting-started/advanced_tutorial.md

```diff
@@ -0,0 +1,305 @@
+
+```py   
+from dlt_plus import current
+from dlt_plus.project.run_context import switch_profile
+
+if __name__ == "__main__":
+    # will show that the default profile is active
+    print(current.project().current_profile)
+    # switch to the tests profile
+    switch_profile("tests")
+    # now the tests profile is active and is merged with the project config
+    print(current.project().current_profile)
+```
+
+## Packaging a project
+
+`dlt+` also has tools to help you package a dlt+ project for distribution. This will make your dlt+ project pip installable and make it easier for you to distribute accross your organization. To start with a dlt+ packaged project, you can supply the package argument to the project init command:
+
+```sh
+dlt project init arrow duckdb --package my_dlt_project
+```
+
+This will provide you with the same project as used in the basic tutorial but places in a module directory named `my_dlt_project` and provided with a simple PEP compatible `pyproject.toml` file. You will also receive a default package `__init__.py` file to make the module accssible from the outside when installed:
+
+```sh
+.
+├── my_dlt_project/       # your project module
+│   ├── __init__.py       # your package entry point
+│   ├── dlt.yml           # the dlt+ project manifest
+│   └── ...               # more project files
+├── .gitignore
+└── pyproject.toml        # the main project manifest
+```
+
+Your `dlt.yml` will look and work exactly the same as in the non-packaged version. The only difference is the module structure and the presence of the `pyproject.toml` file. The pyproject.toml has a special entry point setting that makes the dlt+ project discoverable by dlt+:
+
+```toml
+[project.entry-points.dlt_package]
+dlt-project = "my_project"
+```
+
+You can still run the pipeline the same way as before with the cli commands from the root folder:
+
+```sh
+dlt pipeline my_pipeline run
+```
+
+If you look at the module `__init__.py` file that was created for you, you can see the full interface of your packaged project that the user of your package will have access to. It is very similar to the interface of the `current` interface of the flat project, there is an additional feature that selects the `access` profile by default. It is up to you to adapt this `__init__.py` file to your needs.
+
+### Using the packaged project
+
+Using this project in its default state, let's simulate installing it into a new python project like a data-scientist in your organization might do, to demonstrate how this could be used in practice. For this example we use the uv package manager, but this is also possible with poetry or pip. Let's assume you have been developing your dlt+ packaged project at `/Volumes/my_drive/my_folder/pyproject.toml`. Navigate to a new directory and initialize your project:
```

Comment:
```suggestion
Using this project in its default state, let's simulate installing it into a new python project like a data-scientist in your organization might do, to demonstrate how this could be used in practice. For this example we use the uv package manager, but this is also possible with Poetry or pip. Let's assume you have been developing your dlt+ packaged project at `/Volumes/my_drive/my_folder/pyproject.toml`. Navigate to a new directory and initialize your project:
```